content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -14,11 +14,6 @@ This release cycle, we saw a major shift in ownership of processes around the re
 
 ## Major Themes
 
-### CronJobs Graduate to Stable!
```

Comment:
I don't think this needs to be removed per the issue you've quoted since it graduated to stable during this particular release. Please weigh in @sftim .

Reply:
I agree, this should stay